package.json

```diff
@@ -34,7 +34,9 @@
   },
   "dependencies": {
     "@justeat/pie-design-tokens": "3.0.0",
-    "include-media": "eduardoboucas/include-media#2.0-release"
+    "glob": "8.0.3",
```

Comment:
I was getting a lot of extraneous-dependencies which is why I moved these to regular dependencies. Happy to move back, will just need to add some Eslint overrides to the erroring files

Reply:
What were the errors out of interest?

Reply:
it basically said this should be a regular dependency whenever I required the module

Reply:
to be clear it was an IDE error rather than a compilation one

Reply:
it maybe worth reverting the change then?

Reply:
I'm happy to, just unsure if it's a legit concern as it seems to go against our eslint config

Reply:
If it's not required by projects consuming this package to use it then I'd say it belongs in `devDependencies`.

Reply:
Also, moving `sass` into this section will force consumers to pull that version locally.

Reply:
will move it back now 👍